libsrc/nes/Makefile.inc

```diff
@@ -1,8 +1,8 @@
-../tgi/nes-64-56-2.tgi: ../libwrk/nes/clrscr.o \
-                        ../libwrk/nes/cputc.o  \
-                        ../libwrk/nes/get_tv.o \
-                        ../libwrk/nes/gotoxy.o \
-                        ../libwrk/nes/popa.o   \
-                        ../libwrk/nes/ppu.o    \
-                        ../libwrk/nes/ppubuf.o \
-                        ../libwrk/nes/setcursor.o
+../target/nes/drv/tgi/nes-64-56-2.tgi: ../libwrk/nes/clrscr.o \
+                                   ../libwrk/nes/cputc.o  \
+                                   ../libwrk/nes/get_tv.o \
+                                   ../libwrk/nes/gotoxy.o \
+                                   ../libwrk/nes/popa.o   \
+                                   ../libwrk/nes/ppu.o    \
+                                   ../libwrk/nes/ppubuf.o \
+                                   ../libwrk/nes/setcursor.o
```

Comment:
A regression after some later changes - corrected. If we want to "fold" it differently - let me know.